Dockerfile

```diff
@@ -4,31 +4,20 @@ ARG TARGETOS
 ARG TARGETARCH
 
 WORKDIR /workspace
-# Copy the Go Modules manifests
-COPY go.mod go.mod
-COPY go.sum go.sum
-# cache deps before building and copying source so that we don't need to re-download as much
-# and so that source changes don't invalidate our downloaded layer
-RUN go mod download
-
-# Copy the go source
-COPY cmd/ cmd/
-COPY api/ api/
-COPY internal/ internal/
-COPY pkg/ pkg/
+COPY . .
```

Comment:
This can be a pain when building locally, btw. Your local workspace will almost always be dirty for one reason or another. I'd consider reverting this part of the change and adding the parts you need explicitly. In particular you want the `go mod download` to run early in its own cache layer.

Reply:
OK, I've reverted this change and added only the necessary scripts.